### ISSUE TITLE
Documentation regarding FinTS

### ISSUE THREAD

**Issue by gavu** [firefly-iii/firefly-iii#1836]
I am running Firefly III version 4.7.8

First of all, awesome implementation, great job. It should turn all my python scripts obsolete. 

After importing a few entries from last week, got no errors (I am however curious how transfers are created now, as I had to modify these transactions before importing them to match the transfer-pattern)

Can be automated via CronJob, if yes, where exactly do I store my credentials? Also, is it possible to import multiple accounts?

I checked the documentation and the issue/changelog entry 
https://github.com/firefly-iii/firefly-iii/issues/1159 but had no luck.

The `firefly-iii/config/import.php` contains an `options` array, which, however does not contain a `fints` object, so no luck there either.

 Any pointers?

Thanks in advance!

**Comment by JC5** [comment]
There's not much yet. Currently the process works through the user interface only. Improvements might be tricky because there are no endpoints for me to test on. But looking at your post, I think we can start thinking about:

* Import of transfers. What would that pattern be, and how can the code be improved? What did you have to change to get this fixed?
* Secure credential storage. I can fix a basic CRUD interface somewhere in the preferences, but I am running out of space (in the interface). Might be that I have to implement #1368 before I can start on this (applies to #1831 too for example)
* Command line importing is on the list for other import providers as well.

**Comment by JC5** [comment]
Also tagging @bnw.

**Comment by bnw** [comment]
Regarding transfers:
This just works for me. If I have a transfer A->B and I import via FinTS from both account A and B (in two seperate imports), Firefly will correctly present these two transactions as a single transfer. @JC5 I guess firefly does some matching there based on amount and date?

**Comment by JC5** [comment]
Yes, Firefly III will automatically merge transfers (aka ignore the second one) when you import A>B and then B>A, even when these are imported over different sessions. This will work only if the date, description, amount and accounts are the same.

**Comment by timbuchwaldt** [comment]
👍for automatic CronJob imports.  I host Firefly myself so for example a way of passing a secret via Cron (from a file that is root-only readable) would be great. Otherwise I would love to at least be able to save my FinTS settings and just enter either a password (that decrypts my stored PIN/Credentials) or just enter the FinTS Password.

**Comment by JC5** [comment]
https://github.com/firefly-iii/firefly-iii/issues/1461
